### ISSUE TITLE
help

### ISSUE THREAD

**Issue by guojin1211** [good4488/ZeoGAN#1]
Hello, there is no data set and no running method in this project. Can you share the data set and running method?

**Comment by good4488** [comment]
Hello, here are dataset links for this project.
Training set  :  https://www.dropbox.com/s/0v5pr6km0jasbpl/training.tgz?dl=0
Test set  :  https://www.dropbox.com/s/3qhs70wez5ef9q4/test.tgz?dl=0

(files in dataset)
.grid file  :  lattice parameters
.griddata file  :  methane-zeolite interaction energy  (32x32x32 energy grid)
.O file  :  oxygen positions (32x32x32 Gaussian grid)
.si file  :  silicon positions  (32x32x32 Gaussian grid)

Also, I updated README to share running method.

Thanks!

**Comment by krk-krk-krk** [comment]
Hello good4488.

What software can handle these grid files?
We would appreciate it if you could share an example of the software name.
